_posts/en/newsletters/2024-02-07-newsletter.md

```diff
@@ -0,0 +1,319 @@
+      right, before giving up the idea of free relay protection
+      entirely."  ---Sanders
+
+    - "The mempool as it exists to today doesn’t support an efficient
+      way to calculate “miner score” or incentive compatibility, due to
+      unbounded cluster sizes. [...] One advantage of cluster mempool is
+      being able to calculate things like miner score and incentive
+      compatibility across the mempool. Similarly, one advantage of v3
+      is being able to do this before cluster mempool because of
+      restricted topology. Before people took on the challenge of
+      designing and implementing cluster mempool, I had been framing v3
+      as “cluster limits” without having to implement cluster limits, as
+      it’s one of the only ways to codify a cluster limit (count=2)
+      using existing package limits (ancestors=2, descendants=2. Once
+      you go up to 3, you can have infinite clusters again). Another
+      advantage of v3 is that it helps unblock cluster mempool, which is
+      in my opinion a no-brainer.  In summary, I don’t think the
+      One-Shot Replace by Feerate proposal works (i.e. doesn’t have a
+      free relay problem and is feasible to implement accurately)."
+      ---Zhao
+
+  The separate discussions were not reconciled as of this writing.
+  Peter Todd has released an experimental [implementation][libre relay]
+  of the replace by feerate rules.
+
+- **Bitcoin-Dev mailing list migration update:** as of this writing, the
+  Bitcoin-Dev mailing list is no longer accepting new emails as part of
+  the process of migrating it to a different list server.  Optech will
+  provide an update when the migration is complete.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME:harding to update Tuesday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+- [Bitcoin Core #29189][] deprecates libconsensus.  Libconsensus was an
+  attempt to make Bitcoin Core's consensus logic usable in other
+  software.  However, the library hasn't seen any significant adoption
+  and it has become a burden on maintenance of Bitcoin Core.  The plan
+  is to "not migrate it to CMake and let it end with v27. Any remaining
+  use-cases could be handled in the future by libbitcoinkernel."
+
+- [Bitcoin Core #28956][] removes adjusted time from Bitcoin Core and
```

Comment:
Could link adjusted time to https://bitcoinops.org/en/newsletters/2024/01/10/#bitcoin-core-pr-review-club